pylutron/__init__.py

```diff
@@ -1321,6 +1444,11 @@ def name(self):
     """Returns the name of this area."""
     return self._name
 
+  @property
+  def location(self):
+    """Returns the name of this area."""
```

Comment:
Returns the location of this area which is the name of the parent area or the empty string.